yarn.lock

```diff
@@ -0,0 +1,2 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Do we need to add this file to the repo?

Reply:
I'm surprised this wasn't caught by the gitignore... either way good catch!